src/grt/src/GlobalRouter.cpp

```diff
@@ -304,6 +312,105 @@ void GlobalRouter::estimateRC()
   }
 }
 
+bool GlobalRouter::findWorstSlackNets(float worst_nets_percentage)
+{
+  std::vector<float> slack_values;
+  std::map<odb::dbNet*, float> slack_per_net;
+
+  rsz_->resizeSlackPreamble();
+  rsz_->findResizeSlacks();
+
+  std::vector<odb::dbNet*> worst_slack_nets = rsz_->resizeWorstSlackDbNets();
```

Comment:
Why are you using the virtual resizer API grt - resizing should already have happened.   Is this for inside of gpl?